frontend/src/pages/ProjectDetails.tsx

```diff
@@ -41,36 +62,256 @@ const ProjectDetailsPage = () => {
+                      {leader}
+                    </span>
+                  ))}
+                </span>
+              </p>
+            </div>
+            <p>
+              <strong>Last Updated:</strong> {formatDate(project.idx_updated_at)}
+            </p>
+            <p>
+              <strong>URL:</strong>{' '}
+              <a
+                href={project.idx_url}
+                className="text-blue-600 hover:underline dark:text-blue-400"
+              >
+                {project.idx_url}
+              </a>
+            </p>
+          </div>
+
+          <div className="rounded-lg bg-gray-100 p-6 shadow-md dark:bg-gray-800">
+            <h2 className="mb-4 text-2xl font-semibold">Statistics</h2>
+            <div className="mb-2 flex items-center">
+              <FontAwesomeIcon icon={faUsers} className="mr-2" />{' '}
+              <span>{project.idx_contributors_count} Contributors</span>
+            </div>
+            <div className="mb-2 flex items-center">
+              <FontAwesomeIcon icon={faCodeFork} className="mr-2" />{' '}
+              <span>{project.idx_forks_count} Forks</span>
+            </div>
+            <div className="mb-2 flex items-center">
+              <FontAwesomeIcon icon={faStar} className="mr-2" />{' '}
+              <span>{project.idx_stars_count} Stars</span>
+            </div>
+            <div className="mb-2 flex items-center">
+              <FontAwesomeIcon icon={faBook} className="mr-2" />{' '}
+              <span>{project.idx_issues_count} Issues</span>
+            </div>
+            <div className="flex items-center">
+              <FontAwesomeIcon icon={faCode} className="mr-2" />{' '}
+              <span>{project.idx_repositories_count} Repositories</span>
+            </div>
+          </div>
+        </div>
+
+        <div className="mb-8 rounded-lg bg-gray-100 p-6 shadow-md dark:bg-gray-800">
+          <h2 className="mb-4 text-2xl font-semibold">Summary</h2>
+          <p>{project.idx_summary}</p>
+        </div>
+
+        <div className="mb-8 grid grid-cols-1 gap-6 md:grid-cols-2">
+          <div className="rounded-lg bg-gray-100 p-6 shadow-md dark:bg-gray-800">
+            <h2 className="mb-4 text-2xl font-semibold">Languages</h2>
+            <div className="flex flex-wrap gap-2">
+              {(showAllLanguages ? project.idx_languages : project.idx_languages.slice(0, 10)).map(
+                (lang, index) => (
+                  <span
+                    key={index}
+                    className="rounded-full bg-blue-500 px-2 py-1 text-sm text-white"
+                  >
+                    {lang}
+                  </span>
+                )
+              )}
+            </div>
+            {project.idx_languages.length > 10 && (
+              <button
+                onClick={toggleLanguages}
+                className="mt-4 flex items-center text-blue-600 hover:underline dark:text-blue-400"
```

Comment:
We also have a custom color for links. It's not standard blue from tailwind library. It's a muted blue. I believe it was `#1d7bd7` but you should check `Card` links to be sure 👍🏼